packages/ckeditor5-image/src/imageinsert/imageinsertviaurlui.ts

```diff
@@ -46,28 +70,35 @@ export default class ImageInsertViaUrlUI extends Plugin {
 	public afterInit(): void {
 		this._imageInsertUI = this.editor.plugins.get( 'ImageInsertUI' );
 
+		const buttonViewCreator = () => {
+			const button = this.editor.ui.componentFactory.create( 'uploadUrl' ) as ButtonView;
+
+			button.withText = true;
+
+			return button;
+		};
+
 		this._imageInsertUI.registerIntegration( {
 			name: 'url',
 			observable: () => this.editor.commands.get( 'insertImage' )!,
-			requiresForm: true,
-			buttonViewCreator: isOnlyOne => this._createInsertUrlButton( isOnlyOne ),
-			formViewCreator: isOnlyOne => this._createInsertUrlView( isOnlyOne )
+			requiresForm: false,
+			buttonViewCreator,
```

Comment:
This is probably incorrect :thinking:. If insert via URL is the only available option, then it will be displayed in the toolbar with text, right? Which is wrong.

Reply:
Also, I just realized that `uploadUrl` is rather a bad name. First question: upload what? Second: we don't upload anything really.

The other components we have are called:

* `insertImage`
* `uploadImage`

`urlImage` sounds quite bad so I propose `insertImageUrl` (and ofc `menuBar:insertImageUrl`).

Reply:
Also please add info about this component to metadata file.